### DISCUSSION TITLE
Possible to remove supplier part while retaining historical info?

### DISCUSSION THREAD

**Discussion by LastEdit** [inventree/InvenTree#7465] [Q&A]
Sometimes a supplier will stop selling a part and I would like it to be removed from the order drop down options. Can you remove the supplier part from showing when buying parts, while still keeping the info in the purchase order / stock intact? I did a simple test of removing a supplier part after completing an order and it removed the line items from the order

**Comment by SchrodingersGat** [comment]
You can disable a supplier part in this instance:

https://docs.inventree.org/en/latest/order/company/#disable-supplier-part

**Reply by LastEdit** [reply]
Wow, thank you. Already ahead of it. This is currently only in the new UI correct?

**Reply by SchrodingersGat** [reply]
It looks like this feature is only in the new interface, yeah
